backend/oasst_backend/models/message.py

```diff
@@ -6,6 +6,7 @@
 import sqlalchemy as sa
 import sqlalchemy.dialects.postgresql as pg
 from oasst_backend.models.db_payload import MessagePayload
+from oasst_backend.models.user import User
```

Comment:
can also be imported from modes (omitting .user)

Reply:
Unfortunately, it causes circular import error
![image](https://user-images.githubusercontent.com/33456881/224137487-ed757748-0688-4130-ad50-3dbee647890d.png)

Reply:
ah, ok, we are in the Message model file ;-) ..